src/components/large-countdown-clock/LargeCountdownClock.js

```diff
@@ -0,0 +1,254 @@
+import useCountdown from '@bradgarropy/use-countdown';
+import React, { useState } from 'react';
+import ReactNativeHapticFeedback from 'react-native-haptic-feedback';
+import Animated, {
+  Easing,
+  useAnimatedProps,
+  useSharedValue,
+  withSpring,
+  withTiming,
+} from 'react-native-reanimated';
+import { Circle, Defs, G, RadialGradient, Stop, Svg } from 'react-native-svg';
+import CheckmarkAnimation from '../animations/CheckmarkAnimation';
+import { Flex } from '../layout';
+import { Text } from '../text';
```

Comment:
Prefer design system `Text`

Reply:
the setup for this animated component is too custom to apply the design system imo. gonna leave it as is